app/routes/__frontend/resources/virtual-coffee-handbook/guides-to-virtual-coffee/what-to-expect-in-virtual-coffee.mdx

```diff
@@ -0,0 +1,277 @@
+---
+meta:
+  title: What to Expect in Virtual Coffee
+  description: Things happen in Virtual Coffee (with how-to instructions).
+hero:
+  Hero: UndrawRemoteMeeting
+tags: [memberresources]
+order: 2
+---
+
+import LeadText from '~/components/content/LeadText';
+import TextContainer from '~/components/content/TextContainer';
+import { Link } from '@remix-run/react';
+
+<TextContainer background="light" showBackToTopLink={false}>
+
+<LeadText>
+
+We get many questions like "What to expect once we join Virtual Coffee?" "Where's the link to today's Coffee?" "How to join a Virtual Coffee event?" This article serves as a guide to answer these questions!
+
+</LeadText>
+
+</TextContainer>
+
+## Table of Contents
+
+<TextContainer>
+
+---
+
```

Comment:
Link to the glossary first maybe?

Reply:
Would it be better to link it in line 35 instead of the title (`h2`)?

We have [Coffee](https://deploy-preview-895--virtual-coffee-io.netlify.app/resources/virtual-coffee-handbook/guides-to-virtual-coffee/glossary#coffees) on **Tuesdays at 9:00 AM ET** and **Thursdays at 12:00 PM ET** that last an hour.

Reply:
yes! sorry didn't mean to imply put it in the heading. That's perfect!